### PR TITLE
Smarter issues

### PR DESCRIPTION
**Pull request by BeroBurny**
Reduce spamming issues

### CODE REVIEW

File: .github/workflows/cron.yml [hunk 1]
```diff
@@ -23,21 +23,28 @@ jobs:
...
 
       - name: Tests
+        id: tests
         run: xvfb-run --auto-servernum yarn run test
 
-      - name: Submit issue
+      - name: Check if issue already exists
         if: failure()
-        uses: imjohnbo/issue-bot@3daae12aa54d38685d7ff8459fc8a2aee8cea98b
+        uses: nickderobertis/check-if-issue-exists-action@master
```
Comment (mpetrunic) [inline .github/workflows/cron.yml:31]: lock specific commit

**Comment by mpetrunic** [comment]
@BeroBurny ci failing?

**Comment by BeroBurny** [comment]
> @BeroBurny ci failing?

After MetaMask release v10.2.0 they broke adding network over UI, and that part of tests are failing.

**Comment by mpetrunic** [comment]
> > @BeroBurny ci failing?
> 
> After MetaMask release v10.2.0 they broke adding network over UI, and that part of tests are failing.

should we fix that first?

**Comment by BeroBurny** [comment]
> > > @BeroBurny ci failing?
> > 
> > 
> > After MetaMask release v10.2.0 they broke adding network over UI, and that part of tests are failing.
> 
> should we fix that first?

As I'm aware we cannot fix it in the way the app is working, properly hacking whit an idea as in #35 will fix it.
For now, the best solution is to ignore this version.

**Pull request merged**